apis/server/container_bridge.go

```diff
@@ -309,6 +309,13 @@ func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *
 		}
 	}
 
+	if len(meta.Mounts) > 0 {
```

Comment:
I do not think we need this judgement, the range will ignore the situation of length 0 automatically.

Reply:
ok

Reply:
ok